packages/core/src/browser/saveable.ts

```diff
@@ -87,10 +87,16 @@ export namespace Saveable {
         return !!getDirty(arg);
     }
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    export async function save(arg: any): Promise<void> {
+    export async function save(arg: any, options?: SaveOptions): Promise<void> {
         const saveable = get(arg);
+        /**
+         * skipFormatting is true if:
+         * - The function receives skipFormattingOnSave flag from 'options' object (ie: from command `saveWithoutFormatting`).
+         * - The file is not dirty and performing save in multiple files (saveAll command).
+         */
```

Comment:
I think the documentation can be simplified, if not removed as the use-case is straightforward.

```
// Formatting should be skipped when explicitly requested, or when performing multiple saves (ex: `Save All`).
```